model/itemTab.cc

```diff
@@ -244,35 +246,38 @@ namespace minsky
                         varAttribVals.push_back(to_string(v->sliderMax));
                         varAttribVals.push_back(to_string(v->value()));
                     
-                        for (auto& i:varAttrib) 
-                          {
-                            cairo_move_to(cairo,x,y-1.5*rowHeight);                    
-                            pango.setMarkup(i);
-                            pango.show();                  
-                            colWidth=std::max(colWidth,5+pango.width());  
-                            x+=colWidth;	
-                            colLeftMargin[iC].push_back(x);                        				    
-                          }
+                        if (it==itemVector[0]) {
```

Comment:
OK, because itemVector no doubt has unique elements, but better to compare the address (&it==&itemVector[0]).

Reply:
Done, thanks :)